mailpoet/lib/Segments/SegmentsRepository.php

```diff
@@ -172,12 +175,19 @@ public function createOrUpdate(
     return $segment;
   }
 
-  public function bulkDelete(array $ids, $type = SegmentEntity::TYPE_DEFAULT) {
+  /**
+   * @param array  $ids   Array of segment ids to delete
+   * @param string $type  Type of segment to delete
+   *
+   * @return int Number of segments deleted
+   */
+  public function bulkDelete(array $ids, $type = SegmentEntity::TYPE_DEFAULT): int {
```

Comment:
I would just add `string $type=SegmentEntity::TYPE_DEFAULT` but all those comments are also just personal favorites right now :smile:

Reply:
I guess I got carried away 😅 
I have moved the types to the signature when possible and removed the comments. It is more consistent with the rest of the codebase.